### DISCUSSION TITLE
Cuy University semester 4 - sesi 8

### DISCUSSION THREAD

**Discussion by vikiadzan** [deaafrizal/forum#31] [Troubleshooting Room]
Punten, akang-akang,teteh saya lagi ngikuti sesi 8  , pas coba Authorize github muncul pesan **Try signing in with a different account.** itu gimana ya, tolong solusinya

`import modul`
cara1
import { authOption } from "@/app/api/auth/[...nextauth]/route"
cara2
import { authOption } from "../../app/api/auth/[...nextauth]/route"

pada sesi 7 saya masih bisa menggunkan impor _cara1_
pada pertengahan sesi 8 tidak bisa lagi menggunakan _cara 1_ dan harus menggunakan _cara2_

Mohon solusi nya

**Comment by deaafrizal** [comment]
harusnya message itu bukan dari masalah import route sih, coba generate github app baru, ganti env ke yang baru. apakah aman atau gak tanpa ubah code projeknya.

**Reply by vikiadzan** [reply]
Aman bang, ada yang typo
tengkyu bang
